<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiverSettingsDlg</class>
 <widget class="QDialog" name="ReceiverSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="TabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="sync">
      <attribute name="title">
       <string>Sync</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Channel Estimation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Frequency Interpolation</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="RadioButtonFreqWiener">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Wiener</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButtonFreqLinear">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Linear</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButtonFreqDFT">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>DFT Zero Pad.</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Time Interpolation</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="RadioButtonTiWiener">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Wiener</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButtonTiLinear">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Linear</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Time Sync Tracking</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="RadioButtonTiSyncEnergy">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Guard Energy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButtonTiSyncFirstPeak">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>First Peak</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="misc">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Interferer Rejection</string>
         </property>
         <widget class="QCheckBox" name="CheckBoxRecFilter">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>153</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>&amp;Bandpass Filter</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="CheckBoxModiMetric">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>153</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Modified &amp;Metrics</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBoxFlipSpec">
         <property name="text">
          <string>&amp;Flip Input Spectrum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>MLC</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="TextNumOfIterations">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Number Of Iterations</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="SliderNoOfIterations">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>6</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logging">
      <attribute name="title">
       <string>Logging</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QCheckBox" name="CheckBoxWriteLog">
         <property name="text">
          <string>&amp;Log To File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="CheckBoxLogLatLng">
         <property name="text">
          <string>Log &amp;Position Data</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="CheckBoxLogSigStr">
         <property name="text">
          <string>Log Receive Level</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="TextStartDelay">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Start Delay</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabel1_2">
           <property name="text">
            <string>0 s</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="SliderLogStartDelay">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBothSides</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="TextLabel2">
           <property name="text">
            <string>120 s</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Rig">
      <attribute name="title">
       <string>Rig</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0" rowspan="5">
        <widget class="QGroupBox" name="groupBoxRigModes">
         <property name="title">
          <string>Set Rig For</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="checkBoxDRM">
            <property name="text">
             <string>DRM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxAM">
            <property name="text">
             <string>AM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxFM">
            <property name="text">
             <string>FM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxUSB">
            <property name="text">
             <string>AM USB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxLSB">
            <property name="text">
             <string>AM LSB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxCW">
            <property name="text">
             <string>CW</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxNBFM">
            <property name="text">
             <string>NB FM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUseRigForAllModes">
            <property name="text">
             <string>All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Rig</string>
           </property>
           <property name="buddy">
            <cstring>treeViewRig</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="treeViewRig"/>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxRigPort</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QComboBox" name="comboBoxRigPort">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>S-Meter</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QwtThermo" name="ProgrSigStrength">
            <property name="alarmColor">
             <color>
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </property>
            <property name="alarmEnabled">
             <bool>true</bool>
            </property>
            <property name="scalePosition">
             <enum>QwtThermo::TopScale</enum>
            </property>
            <property name="fillColor">
             <color>
              <red>0</red>
              <green>190</green>
              <blue>0</blue>
             </color>
            </property>
            <property name="maxValue">
             <double>60.000000000000000</double>
            </property>
            <property name="minValue">
             <double>-60.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CheckBoxEnableSMeter">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButtonTestRig">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>result</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="pushButtonApplyRig">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gps">
      <attribute name="title">
       <string>GPS</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxReceiverLocation">
         <property name="title">
          <string>Receiver Location</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="TextLatitude">
            <property name="text">
             <string>Latitude   </string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="TextLabel1_3">
            <property name="text">
             <string>Degrees</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="TextLabel2_3">
            <property name="text">
             <string>Minutes</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="ComboBoxNS">
            <property name="maxCount">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>N</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>S</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="LineEditLatDegrees"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="LineEditLatMinutes"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TextLongitude">
            <property name="text">
             <string>Longitude</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="LineEditLngDegrees"/>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="LineEditLngMinutes"/>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="ComboBoxEW">
            <property name="maxCount">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>E</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>W</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxGPS">
         <property name="title">
          <string>GPS</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="CheckBoxUseGPS">
              <property name="text">
               <string>Use GPS</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="LineEditGPSPort"/>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="LineEditGPSHost"/>
            </item>
            <item row="1" column="0">
             <spacer name="Spacer4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="3">
             <spacer name="Spacer5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="GPSHost">
              <property name="text">
               <string>Host</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="GPSPort">
              <property name="text">
               <string>Port</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <spacer name="Spacer6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="CheckBoxDisplayGPS">
            <property name="text">
             <string>Display GPS Data</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="Spacer6_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonClose">
     <property name="windowTitle">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Close</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LineEditLatDegrees</tabstop>
  <tabstop>LineEditLatMinutes</tabstop>
  <tabstop>ComboBoxNS</tabstop>
  <tabstop>LineEditLngDegrees</tabstop>
  <tabstop>LineEditLngMinutes</tabstop>
  <tabstop>ComboBoxEW</tabstop>
  <tabstop>CheckBoxUseGPS</tabstop>
  <tabstop>LineEditGPSHost</tabstop>
  <tabstop>LineEditGPSPort</tabstop>
  <tabstop>CheckBoxRecFilter</tabstop>
  <tabstop>CheckBoxModiMetric</tabstop>
  <tabstop>CheckBoxFlipSpec</tabstop>
  <tabstop>CheckBoxWriteLog</tabstop>
  <tabstop>CheckBoxLogLatLng</tabstop>
  <tabstop>CheckBoxLogSigStr</tabstop>
  <tabstop>SliderLogStartDelay</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
